config/project.yaml

```diff
@@ -278,6 +278,11 @@ projects:
             ica_workflow_version_name: 3.8.4
             modification_time: 2021-09-15T00:21:47UTC
             run_instances: []
+          - name: 3.9.3
+            path: 3.9.3/dragen-umi__3.9.3.cwl
+            ica_workflow_version_name: 3.9.3
+            modification_time: 2021-09-28T01:27:06UTC
```

Comment:
This time stamp is out of date. I think with both of our experiences it may actually stem from this issue here https://github.com/umccr-illumina/cwl-iap/issues/132 and is related to the patch that was used to resolve the issue. Can raise on the illumina call next week.  

I also checked with 
```
$ diff <(cwltool --pack tools/dragen-umi/3.9.3/dragen-umi__3.9.3.cwl 2>/dev/null | jq -r) <(ica workflows versions get wfl.834780d221224322aa6fbe8e5bf15f35 3.9.3 -o json | jq -r '.definition | fromjson')
```
No edits had been made to the CWL tool AND the cwltool packed md5sum matched that in tool.yaml.  

In the meantime I've made a tiny edit on the cwl file, run `cwl-ica tool-sync` with `--force`, rolled back the edit and then re-run `cwl-ica tool-sync`, so now the timestamp is in date at least and future modifications to the file won't need `--force`.